andino_firmware/platformio.ini

```diff
@@ -14,3 +14,10 @@ board = uno
 framework = arduino
 build_flags =
   -Wall -Wextra
+
+[env:nanoatmega328]
```

Comment:
Since `board` is the only field that differs between both configs, let's create a base config for atmelavr platforms and reuse it. I suggest checking PlatformIO's [extends](https://docs.platformio.org/en/latest/projectconf/sections/env/options/advanced/extends.html) feature to do so.

Reply:
Fixed in [0a7f2d2](https://github.com/Ekumen-OS/andino/pull/195/commits/0a7f2d2200d8c10246b80200f20cf7bc37117d44)

PTAL.

Reply:
Looks great, thank you!